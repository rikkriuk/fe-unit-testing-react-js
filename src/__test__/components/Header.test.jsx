import { render } from "@testing-library/react";
import Header from "../../assets/components/Header";

describe("Header Component", () => {
   test("renders header", () => {
      render(<Header />);
      expect(document.querySelector("h1")).toHaveTextContent("Welcome to Header");
   })

   test("match snapshot", () => {
      const { container } = render(<Header />);
      expect(container).toMatchSnapshot();
   })
})