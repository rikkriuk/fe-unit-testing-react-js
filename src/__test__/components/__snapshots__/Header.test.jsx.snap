// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component match snapshot 1`] = `
<div>
  <h1>
    Welcome to Header
  </h1>
</div>
`;
